release-notes/release-notes/5.8-release-notes.md

```diff
@@ -57,6 +57,8 @@ You can now pull specific data from REST API response headers using .NET regex c
 - We fixed a bug in CQL on PostgreSQL that caused the `DATEADD` function to truncate input dates down to `DAY` precision. Now, you can expect more accurate date manipulations without losing finer time details.
 - We fixed an issue where the UI failed to display Batch Data Sync results and instead showed a generic exception message. The jobs tab in the UI now opens without any API failure appearing in the browser's network console.
 - We resolved an issue that caused large batch delta syncs to fail in Cinchy. 
+- We fixed the implementation of `DataPollingConcurrencyIndex`. Additionally, we added logging around the acquire and release processes to enhance monitoring.
```

Comment:
CIN-06566

Reply:
Maybe remove acquire and release from here. Something like:
we added additional logging in Data Polling Listener to enhance monitoring.

Reply:
Incorporated.